### PR TITLE
update authors list

### PR DESCRIPTION
**Pull request by hartig**
This PR extends the authors list before creating our first working draft.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/pull/96.html" title="Last updated on Feb 16, 2021, 7:32 AM UTC (fb396a0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/96/9296a32...fb396a0.html" title="Last updated on Feb 16, 2021, 7:32 AM UTC (fb396a0)">Diff</a>

### CODE REVIEW

**Comment by hartig** [comment]
Just waiting for @TallTed 's response.

File: rdf-star-cg-spec.html [hunk 1]
```diff
@@ -37,6 +41,22 @@
...
+          "company": "TopQuadrant, Inc.",
+        },
+        {
+          "name": "Peter F. Patel-Schneider",
+          "company": "PARC",
+        },
+        {
+          "name": "Andy Seaborne",
+          "company": "Apache Software Foundation",
+        },
```
Comment (TallTed) [inline rdf-star-cg-spec.html:59]:
```suggestion
        },
        {
          "name": "Ted Thibodeau, Jr.",
          "company": "OpenLink Software, Inc.",
```

**Comment by hartig** [comment]
@akuckartz what is your affiliation?

**Comment by hartig** [comment]
@pchampin this PR should be ready now; if you are also happy with it, please merge it and go ahead with publishing the WD

**Comment by hartig** [comment]
@akuckartz do you have an affiliation? We can still add it to the document but you have to let us know **now**

**Comment by akuckartz** [comment]
@hartig There is no relevant affiliation.

**Comment by hartig** [comment]
Okay, thanks.

**Pull request merged**